src/requirements/WalletActivity/WalletActivityForm.tsx

```diff
@@ -103,17 +85,7 @@ const WalletActivityForm = ({
   const chain = useWatch({ name: `${baseFieldPath}.chain` })
   const isEditMode = !!field?.id
 
-  const supportedRequirementTypes = COVALENT_CHAINS.has(chain)
-    ? walletActivityRequirementTypes
-        .filter(({ value }) => !COVALENT_EXCLUDED_TYPES.has(value))
-        .map(
-          ({ value, ...rest }) =>
-            ({
-              ...rest,
-              value: value.replace("ALCHEMY_", "COVALENT_"),
-            } as SelectOption)
-        )
-    : walletActivityRequirementTypes
+  const supportedRequirementTypes = walletActivityRequirementTypes
```

Comment:
This variable is actually redundant, let's just use `walletActivityRequirementTypes` everywhere